Hannah-Sten/TeXiFy-IDEA
# Fix timeout when running system commands

Fix #1680

#### Summary of additions and changes

* Timeout was not used correctly
* Only create subdirs in output dir for non-MiKTeX installations
* \part is not a sectioning command when exam class is used
* Don't crash on extraneous parentheses in log output

#### How to test this pull request

Make bash ask for user input, e.g. in WSL install keychain and put `/usr/bin/keychain --nogui $HOME/.ssh/id_rsa` in `~/.bashrc`, then note that `bash -ic "pdflatex --version"` asks for a password. Then check that the Run config UI does not show WSL anymore, but also does not freeze.
